draft-ietf-dnsop-svcb-https.md

```diff
@@ -1702,6 +1702,240 @@ the authoritative.  Thus, splitting the RR type would double, or in
+    \x00\x02                                           # length: 2 bytes
+    \x00\x35                                           # value
+
+This example has a key that is not registered, its value is unquoted.
+
+    1 foo.example.com key667=hello
+
+    \# 28 (
+    00 01                                              ; priority
+    03 66 6f 6f 07 65 78 61 6d 70 6c 65 03 63 6f 6d 00 ; target
+    02 9b                                              ; key 667
+    00 05                                              ; length 5
+    68 65 6c 6c 6f                                     ; value
+    )
+
+    \x00\x01                                           # priority
+    \x03foo\x07example\x03com\x00                      # target
+    \x02\x9b                                           # key 667
+    \x00\x05                                           # length 5
+    hello                                              # value
+
+This example has a key that is not registered, its value is quoted and contains
+a decimal-escaped character.
+
+    1 foo.example.com key667="hello\210qoo"
+
+    \# 32 (
+    00 01                                              ; priority
+    03 66 6f 6f 07 65 78 61 6d 70 6c 65 03 63 6f 6d 00 ; target
+    02 9b                                              ; key 667
+    00 09                                              ; length 9
+    68 65 6c 6c 6f d2 71 6f 6f                         ; value
+    )
+
+    \x00\x01                                           # priority
+    \x03foo\x07example\x03com\x00                      # target
+    \x02\x9b                                           # key 667
+    \x00\x09                                           # length 9
+    hello\xd2qoo                                       # value
+
+Here, two IPv6 hints are quoted in the presentation format.
+
+    1 foo.example.com ipv6hint="2001:db8::1,2001:db8::53:1"
+
+    \# 55 (
+    00 01                                              ; priority
+    03 66 6f 6f 07 65 78 61 6d 70 6c 65 03 63 6f 6d 00 ; target
+    00 06                                              ; key 6
+    00 20                                              ; length 32
+    20 01 0d b8 00 00 00 00 00 00 00 00 00 00 00 01    ; first address
+    20 01 0d b8 00 00 00 00 00 00 00 00 00 53 00 01    ; second address
+    )
+
+    \x00\x01                                           # priority
+    \x03foo\x07example\x03com\x00                      # target
+    \x00\x06                                           # key 6
+    \x00\x20                                           # length 32
+    \x20\x01\x0d\xb8\x00\x00\x00\x00
+         \x00\x00\x00\x00\x00\x00\x00\x01              # first address
+    \x20\x01\x0d\xb8\x00\x00\x00\x00
+         \x00\x00\x00\x00\x00\x53\x00\x01              # second address
+
+
+This example shows a single IPv6 hint in IPv4 mapped IPv6 presentation
+format.
+
+    1 example.com ipv6hint="2001:db8:ffff:ffff:ffff:ffff:198.51.100.100"
```

Comment:
This may need to done in multi-line format to allow it to fit nicely into .txt format.

Reply:
This _should_ fit the .txt file as it is lined out to be on the exact maximum